src/mon/PGMonitor.cc

```diff
@@ -2061,7 +2061,7 @@ void PGMonitor::get_health(list<pair<health_status_t,string> >& summary,
     if (p->first & PG_STATE_INCOMPLETE)
       note["incomplete"] += p->second;
     if (p->first & PG_STATE_BACKFILL_WAIT)
-      note["backfill"] += p->second;
+      note["waiting for backfill"] += p->second;
```

Comment:
@theanalyst 

should be "backfill-wait" instead? to be consistent with the document.

or, we should be consistent with the naming of "foo-wait", like "recovery_wait". so we should update both the doc and the code:

- s/backfill-wait/backfill_wait/ in doc
- s/"backfill"/"backfill_wait"/ in the line above